Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/ValueListGroup/ByValueListGroup.tsx

```diff
@@ -171,7 +171,6 @@ const ValueListGroups: Application.Types.iByComponent = (props) => {
                         Field={'Items'}
```

Comment:
```suggestion
                        Field={'ItemCount'}
```
Change field to reflect custom view